compiler/src/iree/compiler/Dialect/Stream/Transforms/EmplaceAllocations.cpp

```diff
@@ -98,8 +110,10 @@ static bool tryEmplaceDispatchOp(IREE::Stream::AsyncDispatchOp dispatchOp) {
       targetResult = updateOp.getResult();
       targetResultSize = updateOp.getTargetSize();
     }
-    if (!targetResource)
-      continue;
+    if (!targetResource) {
+      // TODO(#14566): continue if sparse emplacement on multiple results.
```

Comment:
Is the TODO relevant here? This seems like it's just checking whether there is an updateOp users.